intel_extension_for_transformers/neural_chat/pipeline/plugins/retrieval/detector/intent_detection.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
 
     def intent_detection(self, model_name, query):
         """Using the LLM to detect the intent of the user query."""
-        if is_openai_model(model_name):
+        if is_openai_model(model_name) or "http" in model_name:

```

Comment:
shall we wrap a similar function like is_self_deployed_model? then you can check whether http is there.

Reply:
ok, modified.